pimd/pim_cmd.c

```diff
@@ -8813,13 +8817,13 @@ DEFUN (interface_ip_pim_hello,
 
 DEFUN (interface_no_ip_pim_hello,
        interface_no_ip_pim_hello_cmd,
-       "no ip pim hello [(1-180) [(1-630)]]",
+       "no ip pim hello [(1-65535) [(1-65535)]]",
        NO_STR
        IP_STR
        PIM_STR
        IFACE_PIM_HELLO_STR
-       IFACE_PIM_HELLO_TIME_STR
-       IFACE_PIM_HELLO_HOLD_STR)
+       IGNORED_IN_NO_STR
+       IGNORED_IN_NO_STR)
 {
 	char hello_default_timer[3];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
changed to destroy